src/utils/rarityHelpers.jsx

```
@@ -12,20 +12,177 @@ import { Sparkles, Star, Diamond, Circle, Zap, Crown } from 'lucide-react';
+ * RARITY_COLORS. They had already drifted — UserProfile rendered `event` gold
+ * where everything else rendered it orange, and its unknown-rarity fallback was
+ * grey where this file's was gold — which is exactly the class of bug the comment
+ * above was written about. The copies are gone; they all read this now.
+ *
+ * `order` is presentation only: it sorts lists and nothing persists it, so tiers
+ * can be inserted without a migration. `color` is the fill/border/glow value.
+ * `ink` is the same hue lifted to clear WCAG AA as text on the wheel's panels —
+ * see the note in config/constants.js. Where `ink` is absent the fill already
+ * clears the floor and doubles as text.
+ */
+export const RARITY = {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)